### ISSUE TITLE
autoplot fails with multivariate timeseries and new_data in forecast

### ISSUE THREAD

**Issue by TylerGrantSmith** [tidyverts/fabletools#284]
``` r
library(tsibble)
library(fable)
#> Loading required package: fabletools

set.seed(40)
x = rnorm(24, 1)
y = rnorm(24, 30)

xy = data.frame(x,y)
train = as_tsibble(ts(xy, frequency = 12, start = c(2020, 1)), pivot_longer = FALSE)
test  = as_tsibble(ts(xy, frequency = 12, start = c(2022, 1)), pivot_longer = FALSE)

fit <- train %>% model(VAR(vars(x,y) ~ AR(1)))

# works
fc <- forecast(fit)
autoplot(fc)
```

![](https://i.imgur.com/vYVObFE.png)

``` r

# doesn't work
fc <- forecast(fit, new_data = test)
autoplot(fc)
#> Error: Names must be unique.
#> x These names are duplicated:
#>   * "x" at locations 6 and 8.
#>   * "y" at locations 7 and 9.
#> i Use argument `names_repair` to specify repair strategy.
```

<sup>Created on 2020-10-12 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by mitchelloharawild** [comment]
Thanks, fixed in 245d9151234fb8712272034bdcfb7b45c77fcd3b

**Comment by mitchelloharawild** [comment]
Thanks! Should work now:

``` r
library(tsibble)
library(fable)
#> Loading required package: fabletools

set.seed(40)
x = rnorm(24, 1)
y = rnorm(24, 30)

xy = data.frame(x,y)
train = as_tsibble(ts(xy, frequency = 12, start = c(2020, 1)), pivot_longer = FALSE)
test  = as_tsibble(ts(xy, frequency = 12, start = c(2022, 1)), pivot_longer = FALSE)

fit <- train %>% model(VAR(vars(x,y) ~ AR(1)))

# works
fc <- forecast(fit)
autoplot(fc)
```

![](https://i.imgur.com/zNZKmMC.png)

``` r

# doesn't work
fc <- forecast(fit, new_data = test)
autoplot(fc)
```

![](https://i.imgur.com/CEaZgSt.png)

<sup>Created on 2020-10-14 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>
